### DISCUSSION TITLE
How to add dreamcast games?

### DISCUSSION THREAD

**Discussion by D0ngsal** [rg35xx-cfw/Koriki#42] [Q&A]
i have added some dreamcast roms on the dreamcast folder but it wont show on any system.. any fix for that?

**Comment by acmeplus** [comment]
They should show up automatically if you have the right file extensions, so far it should work automatically with .chd and .cdi files.

**Comment by Mrluck23** [comment]
To add Dreamcast games, use specialized emulation software like NullDC or Redream. Download game ROMs legally, configure the emulator, then load the ROM file to play the games.
